src/AspNet.Security.OpenIdConnect.Extensions/OpenIdConnectConstants.cs

```diff
@@ -9,18 +9,41 @@ public static class OpenIdConnectConstants {
         public static class Claims {
             public const string Active = "active";
             public const string Address = "address";
+            public const string AtHash = "at_hash";
+            public const string Birthdate = "birthdate";
             public const string ClientId = "client_id";
             public const string Confidential = "confidential";
+            public const string Email = "email";
```

Comment:
Since ASOS now has these constants, is it worth using them in our own code?

(e.g https://github.com/aspnet-contrib/AspNet.Security.OpenIdConnect.Server/blob/dev/src/AspNet.Security.OpenIdConnect.Server/OpenIdConnectServerHandler.Userinfo.cs#L223)